javaslang/src/main/java/javaslang/collection/Array.java

```diff
@@ -736,7 +736,7 @@ public T get(int index) {
                     list.add(u);
                 }
             }
-            return wrap(toArray(list));
+            return wrap(list.toArray());
```

Comment:
Since we already know this is an arraylist, calling toArray directly avoids creating an IterableWithSize merely to call toArray on it -- probably a micro-optimization, but...